src/en/space-station-14/departments/science/proposals/scientific-research-proposal.md

```diff
@@ -0,0 +1,35 @@
+# Scientific research proposal
+
+| Designers     | Implemented | GitHub Links |
+|---------------|-------------|--------------|
+| chromiumboy   | :x: No      | TBD          |
+
+## Problem statement
+At present the science department faces several key gameplay issues when it comes to unlocking items through scientific research. These include (but are not limited to):
+-	A large number of research projects that can be considered ‘filler’. These projects unlock either very niche or very common place items on the station; a choice between unlocking magboots, hydroponic trays, or buildable research disk terminals is not compelling for the player. 
+-	Due to the high proportion of filler projects at lower research tiers, typically one project will be seen as the ‘optimal’ choice to research (generally either in the industrial or experiment disciplines), leading to many niche items being ignored in the early to mid stages of the shift when they would be most useful, as well as a sense of sameness in technology unlocking despite the research projects being presented at random.
+-	Many research items seem to be under costed with respect to the current ease of points generation. This leads to shifts where all research projects can be readily completed, particularly during longer than normal shifts and / or shifts that have a large research team.
+-	Once all research projects have been completed, there is no longer any compelling reason to conduct any further research.
+
```

Comment:
Also i'd propose to add problem that was leading to original discussion - other departments don't really hope to get somthing very cool from science in mid-round, and endgame is not something you see every round.
I'd propose to something like that:

- Other departments are not relying on sci in any good way at all, which makes it harder to create mechanics, that require assistance from other departments for science. Early research contains most of stuff is pre-mapped stuff, or very minor improvements, not that much desired by majority of players (fancy drill is cool, but not worth staying and yelling 20 minutes on reception).

Other than that i'd propose adding words about sci hoarding resources (which is okay, i guess) and not really giving people stuff - because they consider that would be bad exchange.

- It is occasional problem that sci department is not willing to share items they researched because they feel like giving resources they collected to print items for other people for free is a bad exchange. It seems like it is related to the problem of HOW sci gets such resources - they either get them from other department  (which is not guaranteed) or from maints (as not renewable source!). Overall feeling giving up non-renewable source (from perspective of your department) feels bad.

Reply:
The first point is a good inclusion, but I'm not worried about the second

Reply:
👍